server/src/test/java/org/opensearch/index/store/remote/filecache/FileCacheTests.java

```diff
@@ -34,12 +32,7 @@ public class FileCacheTests extends OpenSearchTestCase {
 
     @Before
     public void init() throws Exception {
-        path = PathTestUtils.createTestPath("FileCacheTests");
-    }
-
-    @After
-    public void clean() {
```

Comment:
Would be great to still clean up after the test, wdyt @andrross ?

Reply:
The `TestRuleTemporaryFilesCleanup` used by `createTempDir()` takes care of that automatically, right?

Reply:
> `TestRuleTemporaryFilesCleanup`

:+1: apologies, I missed where `createTempDir()` comes from